core_ml_modules/language_processing/word.py

```diff
@@ -0,0 +1,32 @@
+class Word:
+    """
+    Word object containing its original form, canonical form (head of the cluster it belongs to), and part-of-speech tag.
```

Comment:
This definition of 'canonical form' doesn't really make sense to me. Clustering could be done in multiple different forms.

Could you point me to a definition elsewhere?

Reply:
I think it's worth pointing out that properties on this object can only be set once